src/full-screen/components/full-screen-button.tsx

```diff
@@ -0,0 +1,31 @@
+import { render } from "enzyme";
+import React, { useEffect, useState } from "react";
+
+import css from "./full-screen-button.scss";
+
+interface IProps {
+  isFullScreen: boolean;
+  handleToggleFullScreen: () => void
+}
+
+export const FullScreenButton: React.FC<IProps> = (props) => {
+  const {isFullScreen, handleToggleFullScreen} = props;
+  const [isHidden, setIsHidden] = useState(false);
+  useEffect(() => {
+    setTimeout(() => {
+      setIsHidden(true);
+    }, 4000);
+  });
+
+  return (
+    <div className={css.fullScreenToggle}>
+      <div id="fullScreenHelp" className={`${css.fullScreenHelp} ${isHidden? css.hidden : ""}`} >
+        Click here to enter/exit fullscreen →
+      </div>
+      <div>
+        <button className={`${css.fullScreenIcon} ${isFullScreen? css.fullscreen : ""}`}
+           onClick={handleToggleFullScreen} />
```

Comment:
Not a big deal either way, but does the `<button>` need to be inside the `<div>`?